### PR TITLE
Search for renaming file in same folder as config file

### PR DESCRIPTION
**Pull request by Manu06D**
Context : 
Having several efpt.config.json on a project, forced to have only one renaming file. 
Also, can't share renaming setup between efpt.config.json cause renaming file is getting purged with only matching config.

-- efpt.renaming.json
-- Base1
---- efpt.config.json
-- Base 2
---- efpt.config.json

Solution :
Look for renaming file on same folder than the config file. 

-- Base1
---- efpt.config.json
---- efpt.renaming.json
-- Base 2
---- efpt.config.json
---- efpt.renaming.json

### CODE REVIEW

**Comment by ErikEJ** [comment]
Have you tested with custom named files?

File: src/GUI/EFCorePowerTools/Extensions/ProjectExtensions.cs [hunk 1]
```diff
@@ -79,15 +79,24 @@ public static List<string> GetConfigFiles(this Project project)
             return result.OrderBy(s => s).ToList();
         }
 
-        public static string GetRenamingPath(this Project project)
+        public static string GetRenamingPath(this Project project, string optionsPath)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            var projectPath = project.Properties.Item("FullPath")?.Value.ToString();
+            string projectPath;
```
Comment (ErikEJ) [inline src/GUI/EFCorePowerTools/Extensions/ProjectExtensions.cs:86]: Rename to "renamingPath" ?

**Review by ErikEJ — CHANGES_REQUESTED** [review]

**Comment by Manu06D** [comment]
> Have you tested with custom named files?

Hi Erik, I'm not aware about such features ^^ I will check later .. 

Thanks

File: src/GUI/EFCorePowerTools/Extensions/ProjectExtensions.cs (hunk 1 above)
Comment (Manu06D) [reply src/GUI/EFCorePowerTools/Extensions/ProjectExtensions.cs]: done @ErikEJ  !

I originally kept it in order to respect previous code :)

**Comment by Manu06D** [comment]
> Have you tested with custom named files?

Yep, working fine  with config file : efpt*.config.json.

Btw, the code take the path of the config file so if you have 2 differents config files in the same folder, you'll still have the issue of having one renaming file. I parsed the config filename to get the custom pattern and look for appropriate renaming file.

efpt.XXXconfig.json will look for efpt.XXXrenaming.json rename file

**Comment by ErikEJ** [comment]
The pattern is actually efpt.xxxx.config.json

**Review by ErikEJ — CHANGES_REQUESTED** [review]
A regex is not needed, and hard to maintain/understand. 
Just look for starts with efpt.  and ends with .config.json ...

**Comment by Manu06D** [comment]
I reverted the regexp part cause I notice that the matching of renaming file with custom config filename is done in method TryRead on CustomNameOptionsExtensions.

Everything is working fine (tested with multiple several folder / root folder / custom naming)

**Review by ErikEJ — APPROVED** [review]

**Comment by ErikEJ** [comment]
Thanks!

**Comment by Manu06D** [comment]
Thanks Erik ! :) Just watching the youtube video (a little bit late)

**Pull request merged**